README.adoc

```diff
@@ -190,16 +191,15 @@ Create a class to externalize your container configurations.
 `system/src/test/java/it/io/openliberty/guides/system/AppContainerConfig.java`
 ----
 
+The [hotspot=AppContainerConfig file=1]`AppContainerConfig` class externalizes test container setup and configuration, so
```

Comment:
Should "test container" also be changed here to "Testcontainer"?

Reply:
In this case, it's referring to the Kafka Testcontainer as well as the application test container so I generalized both under test container.